trunk/SUAVE/Components/Energy/Storages/Batteries/Constant_Mass/Lithium_Ion.py

```diff
@@ -1,33 +0,0 @@
-## @ingroup Components-Energy-Storages-Batteries-Constant_Mass
```

Comment:
I'll be pushing a few changes that bring this file back as a general class that the other chemistries become a sub-class of

Reply:
Passing the battery chemistry as an argument removed the need for subclasses as the user will only call Lithium_Ion(...) for all options.